MAUI/NumericEntry/Overview.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: About .NET MAUI NumericEntry control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI NumericEntry (SfNumberBox) control, its features, and more.
+platform: maui
+control: SfNumericEntry
+documentation: ug
+---
+
+# .NET MAUI NumericEntry Overview
+
+The `.NET MAUI NumericEntry` control provides an intuitive and advanced editor control that allows you to enter numeric values in currency, percent, decimal formats and much more. It validates the user input independent of the custom format applied. It also validates the input when the focus is lost or the `Enter` key is pressed in `NumericEntry` control. It supports showing watermark text in the editor.
```

Comment:
Replace the Overview page with Whatsnew content

Reply:
Replaced successfully.